docs/use_cases/argocd-gates-using-kubescape.md

```diff
@@ -0,0 +1,63 @@
+#### **Scenario**
+
+A DevOps/Platform team is using ArgoCD for Kubernetes orchestration. They seek to embed security into the workflow to ensure every deployment is secure and compliant with industry benchmarks. To achieve this, Kubescape is chosen to implement security gates within their ArgoCD pipelines. Thus, automatically preventing delivery of any non-compliant deployments.
```

Comment:
I think this scenario section is great, my only request here is to make it less specific on the user side (instead of "DevOps/Platform team is using ArgoCD..." use wording like "When users are using ArgoCD for Kubernetes orchestration and they seek to embed security into..."